client/driver/java.go

```diff
@@ -239,28 +239,16 @@ func (d *JavaDriver) Start(ctx *ExecContext, task *structs.Task) (*StartResponse
 
 	pluginLogFile := filepath.Join(ctx.TaskDir.Dir, "executor.out")
 	executorConfig := &dstructs.ExecutorConfig{
-		LogFile:  pluginLogFile,
-		LogLevel: d.config.LogLevel,
+		LogFile:     pluginLogFile,
+		LogLevel:    d.config.LogLevel,
+		FSIsolation: true,
```

Comment:
Need to implement NewExecutorWithIsolation func on non windows and non linux OSes.

Reply:
done in executor_basic.go